ccpage171.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Add an attribute called number_served with a default value of 0.
+# Create an instance called restaurant from this class.
+# Add a method called set_number_served() that lets you set the number of customers that have been served.
+# Add a method called increment_number_served() that lets you increment the number of customers who’ve been served.
+
+class Restaurant():
+    """An attempt to model a restaurant"""
+    def __init__(self, restuaruant_name, restaurant_cuisine):
+        """Initialize the attributes"""
+        self.restuaruant_name = restuaruant_name
+        self.restuaruant_cuisine = restaurant_cuisine
+        self.number_served = 5
+    
+    def describe_restaruant(self):
+        """Write a variable that returns a summary of the restaurant"""
+        restaurant_details = self.restuaruant_name.title() + " serves " + self.restuaruant_cuisine.title() + " food."
+        return restaurant_details
+
+    # Add a method so I can set the number of guests served
+    def set_number_served(self, guests):
+        """Determine the number of people served at the restaurant"""
+        self.number_served = guests
+
+    # Add a method to increment the total number of people served
+    def increment_number_served(self, incremented_guests):
```

Comment:
Nice job here, I think `new_customer_count` may be a better variable name.

Reply:
@paulghaddad I agree. However, the exercise problem told me to use this variable name, probably to distinguish what I'm doing here -- incrementing.